test/mocks.js

```diff
@@ -174,7 +174,8 @@ function spyLog (methods) {
 function mockRequest (data) {
   return {
     app: {
-      acceptLangage: 'en-US'
+      acceptLangage: 'en-US',
```

Comment:
Yeah, sorry.

The fix for this was in the backed-out PR from the last train, I figured it was low priority enough not to warrant a separate fix of its own. Fwiw, I'm pretty certain that none of the tests actually rely on `acceptLanguage` being set (probably stating the obvious there as they don't fail).